mits/automated_emails.py

```diff
@@ -0,0 +1,20 @@
+from mits import *
```

Comment:
We don't currently have the ability to actually accept/decline submissions, but I'll add those emails when I add that functionality.